x-pack/elastic-agent/docs/upgrade-elastic-agent.asciidoc

```diff
@@ -2,31 +2,43 @@
 [role="xpack"]
 = Upgrade {agent}
 
-beta[]
+experimental[]
 
-//TODO: Add instructions for upgrading from Fleet.
+Starting with version 7.10, {fleet} provides upgrade capabilities. You can view
+and select agents that are out of date, and trigger selected agents to download,
+install, and run the new version.
 
-NOTE: This process is not supported for upgrading DEB/RPM packages or Docker
-images.
+This approach simplifies the process of keeping your agents up to date. It also
+saves you time because you don’t need third-party tools or processes to
+manage upgrades.
+
+NOTE: The upgrade feature is not supported for upgrading DEB/RPM packages or
+Docker images. 
+
+//REVIEWERS: Are bulk actions supported in Basic or just Gold?
 
 To upgrade {agent} to a new version:
 
-. Download and extract the installation package for the new version. 
+. In {fleet}, select **Agents**.
 
-. Run the `install` command to upgrade {agent}.
-+
---
-include::{beats-repo-dir}/x-pack/elastic-agent/docs/tab-widgets/upgrade-widget.asciidoc[]
+. Under **Agents**, click **Upgrade available** to see a list of agents
+that you can upgrade.
 
---
+. To upgrade a single agent, choose **Upgrade agent** from the **Actions** menu
+next to the agent you want to upgrade.
 +
-Follow the prompts to confirm that you want to upgrade {agent}. The
-installation program detects the existing installation and begins upgrading
-to the new version.
+The **Upgrade agent** option is grayed out when an upgrade is unavailable, or
+the {kib} version is lower than the agent version.
 
-To confirm that {agent} is upgraded, go to the **Agents** tab in {fleet}. 
+. To upgrade multiple agents, bulk select the agents and click
+**Upgrade agents**.
++
+[role="screenshot"]
+image:images/fleet-agents.png[Upgrade menu for bulk upgrade in {fleet}]
 
-If you run into problems, see <<ingest-management-troubleshooting>>.
+[[upgrade-standalone]]
+== Upgrade standalone agents
 
-// Add Javascript and CSS for tabbed panels
-include::tab-widgets/code.asciidoc[]
+To upgrade a standalone agent running on an edge node, download the new version
+to your host, then run the `install` command to upgrade to the new version. See
+<<elastic-agent-installation>>.
```

Comment:
i'm uncertain if this is right... I imagine it is 'upgrade' we want to cite.  And the user may not need to download the new artifact.  We need @blakerouse to help confirm

Reply:
or @michalpristas maybe.

Reply:
I didn't get a chance to test this with stand-alone Agent yet, sorry.  I tried to set it up but had problems and didn't get far enough to validate this

Reply:
Ok, I'm going to merge this then follow up with testing. I think there are other changes worth getting into the release docs today. I feel Ok about this because we're still in beta.